src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  let totalCost = days * dailyCost;
+
+  if (days >= 7) {
```

Comment:
2. [Checklist] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values
BAD EXAMPLE:
if (numberOfDays >= 7) {
  return basePrice - 50;
}
GOOD EXAMPLE:
const LONG_TERM = 7;
const LONG_TERM_DISCOUNT = 50;

if (numberOfDays >= LONG_TERM) {
  return basePrice - LONG_TERM_DISCOUNT;
}